zathura/zathura.c

```diff
@@ -546,6 +546,49 @@ document_info_open(gpointer data)
   return FALSE;
 }
 
+const char*
+get_formatted_filename(zathura_t* zathura, const char* file_path, bool statusbar)
+{
+  bool basename_only = false;
+  if (statusbar) {
+    girara_setting_get(zathura->ui.session, "window-title-basename", &basename_only);
+  } else {
+    girara_setting_get(zathura->ui.session, "statusbar-basename", &basename_only);
+  }
+
+  if (basename_only == false) {
+
+    bool home_tilde = false;
+    if (statusbar) {
+      girara_setting_get(zathura->ui.session, "statusbar-home-tilde", &home_tilde);
+    } else {
+      girara_setting_get(zathura->ui.session, "window-title-home-tilde", &home_tilde);
+    }
+
+    if (home_tilde) {
+      char *home = getenv("HOME");
```

Comment:
The documentation says: the function returns $HOME if it is set and fallsback to other means if not.

`user` is also documented on the page you linked: the user or `NULL` to get the home directory of the current user.